content/en/notebooks/advanced_analysis/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Analysis Features
+further_reading:
+- link: "/notebooks"
+  tag: "Documentation"
+  text: "Learn more about Notebooks"
+---
+{{< callout url="#" header="Join the Preview!">}} 
```

Comment:
```suggestion
{{< callout header="Join the Preview!">}} 
```

Reply:
We don't have a form for the preview, and this will be GA very soon (in June), should we just say "contact your CSM to get access" or something along these lines?